api/src/main/java/org/openmrs/util/LocaleUtility.java

```diff
@@ -105,14 +105,8 @@ public static Locale getDefaultLocale() {
 	 * @should confirm matching language as compatible
 	 */
 	public static boolean areCompatible(Locale lhs, Locale rhs) {
-		if (lhs.equals(rhs)) {
-			return true;
-		} else if ((("".equals(lhs.getCountry())) || ("".equals(rhs.getCountry())))
-		        && lhs.getLanguage().equals(rhs.getLanguage())) {
-			// no country specified, so language match is good enough
-			return true;
-		}
-		return false;
+		return lhs.equals(rhs) || (("".equals(lhs.getCountry())) || ("".equals(rhs.getCountry()))) && lhs.getLanguage()
```

Comment:
Shouldn't the right hand side of the first || be enclosed in ()?

Reply:
It was simplified automatically using SonarLint plugin so I hope it's good.

Reply:
I've checked it once again and I believe it is simplified properly